tailwind.config.js

```diff
@@ -118,6 +118,225 @@ module.exports = {
+                            borderRadius: "4px",
+                            background: "rgb(var(--theme-color-secondary-50))",
+                            padding: "5px",
+                            margin: "-5px 0",
+                            fontWeight: "400",
+                        },
+                        figcaption: {
+                            marginTop: "8px",
+                            fontStyle: "italic",
+                            fontSize: "14px",
+                            lineHeight: "24px",
+                            fontWeight: "500",
+                            textAlign: "center",
+                            position: "relative",
+                            "&:after": {
+                                content: "''",
+                                position: "absolute",
+                                left: "0",
+                                right: "0",
+                                bottom: "-8px",
+                                maxWidth: "400px",
+                                with: "100%",
+                                margin: "0 auto",
+                                height: "1px",
+                                backgroundColor: "rgb(var(--theme-color-secondary-300))",
+                            },
+                        },
+                        a: {
+                            color: "rgb(var(--theme-color-hint-600))",
+                            textDecoration: "none",
+                            "&:hover": {
+                                color: "rgb(var(--theme-color-hint-800))",
+                                textDecoration: "underline",
+                            },
+                        },
+                        "--tw-prose-body": "rgb(var(--theme-color-secondary-700))",
+                        "--tw-prose-headings": "rgb(var(--theme-color-secondary-700))",
+                        // "--tw-prose-lead": theme("colors.pink[700]"),
+                        "--tw-prose-links": "rgb(var(--theme-color-hint-600))",
+                        // "--tw-prose-bold": theme("colors.pink[900]"),
+                        // "--tw-prose-counters": theme("colors.pink[600]"),
+                        // "--tw-prose-bullets": theme("colors.pink[400]"),
+                        "--tw-prose-hr": "rgb(var(--theme-color-secondary-300))",
+                        // "--tw-prose-quotes": theme("colors.pink[900]"),
+                        // "--tw-prose-quote-borders": theme("colors.pink[300]"),
+                        "--tw-prose-captions": "rgb(var(--theme-color-secondary-700))",
+                        "--tw-prose-code": "rgb(var(--theme-color-secondary-900))",
+                        "--tw-prose-pre-code": "rgb(var(--theme-color-secondary-900))",
+                        "--tw-prose-pre-bg": "rgb(var(--theme-color-secondary-50))",
+                        // "--tw-prose-th-borders": theme("colors.pink[300]"),
```

Comment:
can all the commented lines go? maybe link to the docs that outline them if it's for documentation purposes